dissect/protos/dns.py

```diff
@@ -0,0 +1,575 @@
+
+# py2+3 compat
+
+from __future__ import unicode_literals
+
+try:
```

Comment:
a construct like this probably belongs in dissect/compat.py ( rather than having a separate issue like this in every protocol module ).  Additionally, we should probably have the semantics lean forward toward 3 rather than backward toward 2 ( ie, not use the name xrange ).